alpaca/rest.go

```diff
@@ -1177,3 +1178,29 @@ func unmarshal(resp *http.Response, data interface{}) error {
 
 	return json.Unmarshal(body, data)
 }
+
+type localPlaceOrderRequest PlaceOrderRequest
```

Comment:
Please add a comment for this. It took me a while to figure out why it was needed.

Reply:
added!